codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 70%
+        
```

Comment:
Please have a new line character at the end of the file

Reply:
fixed